docs/alt-installation.md

```diff
@@ -24,6 +24,83 @@ and then restart your shell.
 
 For instructions on how to install `earthly` for CI use, see the [CI integration guide](./ci-integration.md).
 
+### Installing from Earthly repositories
+
+Earthly can be installed for debian and redhat based linux distributions via the earthly deb and rpm repositories.
+
+All of our binaries are signed with our [PGP key](https://pkg.earthly.dev/earthly.pgp); which has the fingerprint:
+
+    5816 B221 3DD1 CEB6 1FC9 52BA B118 5ECA 33F8 EB64
```

Comment:
```suggestion
    `5816 B221 3DD1 CEB6 1FC9 52BA B118 5ECA 33F8 EB64`
```

Reply:
the leading spaces will turn this into a codeblock.